csm/MotorMan.py

```diff
@@ -0,0 +1,25 @@
+class MotorMan():
+
+    def __init__(self):
+        self.orientationX = 0
+        self.orientationY = 0
+
+    def opticalFlowToMotorDeltas(self, optical_flow):
+        """
+        Converts optical flow data (vectors) into deltas motor must
+        turn to re-center camera view.
+        """
+        deltaX, deltaY = 0,0
+
+        pass
+        
+        return deltaX, deltaY
+
+    def adjustOrientation(self, optical_flow):
```

Comment:
I'm a bit confused about the need for this function here. Can you not adjust the orientation with the function above?

Reply:
Agreed, can simply this module